test/salt/test_pillar.sls

```diff
@@ -0,0 +1,8 @@
+/tmp/formula_pillars.txt:
+  file.managed:
+    - contents:
+      {% raw %}
+      - '{% from "saemref/map.jinja" import saemref with context -%}'
+      - '{{ saemref|json }}'
+      {% endraw %}
+    - template: jinja
```

Comment:
This dump the saemref variables extracted from map.jinja (which is different than ``pillar.items``) in a json file which is read and tested in tests.

I could not find a salt command to do the same thing without intermediary file and custom state file :/